### DISCUSSION TITLE
HomeAssistant Container LXC - with Supervisor?

### DISCUSSION THREAD

**Discussion by ChristophCaina** [tteck/Proxmox#650] [Ideas]
Hi, would it be possible to include HomeAssistant Supervisor into the Container LXC Image?
I've just noticed, that there is absolutely no information about updates or similar on the installation...
I understand, that you might need to pull a new image if there's a new core Version available (I think it is?) but even available Updates for HACS aren't highlighted.

In this case, I have to manually implement some kind of "Monitoring" to check for available Updates (generate a persistent notification, etc.)
I don't want Supervisor for additional Addons - although, I just saw a Video, that even this would be possible... (with a Docker image)...

**Comment by tteck** [comment]
No, Home Assistant Supervised, IMO, isn't worth the trouble. The repo offers 3 ways to install Home Assistant.
1) Home Assistant OS
2) Home Assistant Container
3) Home Assistant Core

https://www.home-assistant.io/installation/#compare-installation-methods

**Comment by ChristophCaina** [comment]
Hi, yes - I already know these and for a fiew years I had a HomeAssistant OS running as a VM on proxmox...
But due to the better resource management I wanted to switch to the LXC installation... a Core Installation on LXC was the first thing I've tried several years ago - but it was pretty unstable back then... your containered version seems to be a good alternate option between both other options... I've just missed the notification for Updates, and some other 'benefits' of the supervised installation.

Anyway - thanks for your reply - I am trying to find some other options on how to deal with this :)
